testnet/launcher/docker.go

```diff
@@ -40,37 +42,29 @@ func (t *Testnet) Start() error {
 		return fmt.Errorf("unable to deploy l1 contracts - %w", err)
 	}
 
-	sequencerNodeConfig := node.NewNodeConfig(
-		node.WithNodeName("sequencer"),
-		node.WithNodeType("sequencer"),
-		node.WithGenesis(true),
-		node.WithSGXEnabled(t.cfg.isSGXEnabled),
-		node.WithEnclaveImage(t.cfg.sequencerEnclaveDockerImage),
-		node.WithEnclaveDebug(t.cfg.sequencerEnclaveDebug),
-		node.WithHostImage("testnetobscuronet.azurecr.io/obscuronet/host:latest"),
-		node.WithL1WebsocketURL("ws://eth2network:9000"),
-		node.WithEnclaveWSPort(11000),
-		node.WithHostHTTPPort(80),
-		node.WithHostWSPort(81),
-		node.WithHostP2PPort(15000),
-		node.WithHostPublicP2PAddr("sequencer-host:15000"),
-		node.WithPrivateKey("8ead642ca80dadb0f346a66cd6aa13e08a8ac7b5c6f7578d4bac96f5db01ac99"),
-		node.WithHostID("0x0654D8B60033144D567f25bF41baC1FB0D60F23B"),
-		node.WithSequencerP2PAddr("sequencer-host:15000"),
-		node.WithManagementContractAddress(networkConfig.ManagementContractAddress),
-		node.WithMessageBusContractAddress(networkConfig.MessageBusAddress),
-		node.WithL1Start(networkConfig.L1StartHash),
-		node.WithInMemoryHostDB(true),
-		node.WithDebugNamespaceEnabled(true),
-		node.WithLogLevel(t.cfg.logLevel),
-		node.WithEdgelessDBImage("ghcr.io/edgelesssys/edgelessdb-sgx-4gb:v0.3.2"), // default edgeless db value
-		node.WithL1BeaconUrl("eth2network:12600"),
-	)
+	edgelessDBImage := "ghcr.io/edgelesssys/edgelessdb-sgx-4gb:v0.3.2"
 	if !t.cfg.isSGXEnabled {
```

Comment:
unrelated to your pr. 
But this condition doesn't make much sense.
"If no sgx then use edglessdb" 
Maybe leave a todo so we look into this at some other time